src/Sentry/Platforms/Android/SentrySdk.cs

```diff
@@ -187,6 +188,10 @@ private static void InitSentryAndroidSdk(SentryOptions options)
 
         // Set options for the managed SDK that depend on the Android SDK. (The user will not be able to modify these.)
         options.AddEventProcessor(new AndroidEventProcessor(androidOptions!));
+        if (options.LogCatIntegration != LogCatIntegrationType.None)
+        {
+            options.AddEventProcessor(new LogCatAttachmentEventProcessor(options.LogCatIntegration));
```

Comment:
Not sure why it's not finding, looks good based on reading the PR

Reply:
> Not sure why it's not finding, looks good based on reading the PR

Because `AndroidOptions` are under `options.Android` instead of `options`, I just didn't see it in VS. Rider is mildly better about this but still not great.